decentraland/unity-renderer
# style: info card thumbnail position

This PR polish backpack UI component

1- Increase max size of the VRM texture quality setting.
<img width="200" alt="Screenshot 2023-08-10 at 15 57 40" src="https://github.com/decentraland/unity-renderer/assets/51088292/2a14923e-4bfd-481d-8d62-3d8e710f7e69">

2- Add mask and fix horizontal position of the info card thumbnail
<img width="637" alt="Screenshot 2023-08-10 at 15 57 47" src="https://github.com/decentraland/unity-renderer/assets/51088292/0954cf53-a591-44ec-b377-75a3a079158e">

3- Add and underline to the marketplace word button to make it more obvious that it is interactive
<img width="853" alt="Screenshot 2023-08-10 at 15 57 51" src="https://github.com/decentraland/unity-renderer/assets/51088292/256753ff-7575-49f4-8ac7-59febc9c99e5">

### How to test

- Open this [link](https://play.decentraland.org/?explorer-branch=style/InfoCardThumbnailPosition). Once the world loaded open the backpack pressing the key [I]
1- Check the VRM button texture doesn't look pixelated
2- Open the info card of an item and check the x position is now correct and when the mask is needed is working
3- Search the item "botas doras" to see the search empty state. Check the word "marketplace" is underlined.
